app/assets/v2/js/pages/profile.js

```diff
@@ -0,0 +1,163 @@
+$(document).ready(function() {
+  $('[data-toggle=popover]').popover({
+    html: true,
+    trigger: 'focus',
+    template: '<div class="popover wallet-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+    content: function() {
+      var content = $(this).attr('data-popover-content');
+
+      return $(content).children('.popover-body').html();
```

Comment:
im not crazy about this popover for a use case like this one..

Reply:
Same like below